rocThrust names libc++'s iterator wrapper with libc++'s internal _VSTD macro
when compiling HIP, which libc++ 19 removed; std::__wrap_iter is the same
type in every libc++ version.

--- a/rocthrust/thrust/type_traits/is_contiguous_iterator.h
+++ b/rocthrust/thrust/type_traits/is_contiguous_iterator.h
@@ -141,11 +141,7 @@
 #if defined(_LIBCPP_VERSION)
 template <typename Iterator>
 struct is_libcxx_wrap_iter<
-#  if _LIBCPP_VERSION < 14000 || THRUST_DEVICE_COMPILER == THRUST_DEVICE_COMPILER_HIP
-  _VSTD::__wrap_iter<Iterator>
-#  else
   std::__wrap_iter<Iterator>
-#  endif
   > : true_type
 {};
 #endif
